.github/workflows/images.yaml

```diff
@@ -7,6 +7,28 @@ jobs:
   build-and-push:
     name: Build and push all images
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        include:
+          - name: maker-image
+
+          - name: tester-image
+
+          - name: kube-test-image
+
+          - name: compilers-image
```

Comment:
Some of the images here depend on compilers image, so you can actually update them in one go, as you can get the git tree hash of the compilers image once you have committed the changes locally.